### ISSUE TITLE
Edit Accessibility Guidelines for Tabbing in Content Tree

### ISSUE THREAD

**Issue by enowak1031** [trimble-oss/website-modus.trimble.com#452]
Tabbing should take the user out of the tree into the main content area. Arrows enable the user to navigate through the tree.

This is because we need a way for the user to skip the tree and go straight to the content. Tabbing through the entire tree could be really frustrating.

**Current Guidelines:** 

- “Enter” or “Space” key should expand and collapse items. “Tab” key moves focus to the next focusable element; all focusable elements in the content tree should be included in the page tab sequence. “Shift” + “Tab” moves focus to the previous focusable element.
- Avoid keyboard traps when adding components to the content tree. For example, the user expands a section, but is unable to tab to the next focusable element or to tab out of the content tree panel.
- Although the content tree element passes accessibility testing, content authors are responsible for ensuring the content in the content tree is accessible.

**New Guidelines:** 

- “Enter” or “Space” key should expand and collapse items. “Arrow” keys move focus to the next focusable element; tabbing allows the user to move to the main content on the page, and should not navigate through the content tree allowing the user to skip it. 
- Avoid keyboard traps when adding components to the content tree. For example, the user expands a section, but is unable to navigate to the next focusable element or to tab out of the content tree panel.
- Although the content tree element passes accessibility testing, content authors are responsible for ensuring the content in the content tree is accessible.
